packages/paste-core/components/chat-composer/src/AutoLinkPlugin.tsx

```diff
@@ -0,0 +1,39 @@
+import * as React from 'react';
```

Comment:
Note: This plugin is taken from the Lexical examples: https://github.com/facebook/lexical/blob/main/packages/lexical-playground/src/plugins/AutoLinkPlugin/index.tsx